corehq/apps/locations/fixtures.py

```diff
@@ -182,34 +182,44 @@ def should_sync_flat_fixture(project):
 
 
 def get_all_locations_to_sync(user):
-    if toggles.SYNC_ALL_LOCATIONS.enabled(user.domain):
-        return LocationSet(SQLLocation.active_objects.filter(domain=user.domain))
-    else:
-        all_locations = set()
-
-        user_locations = set(user.get_sql_locations(user.domain))
-        for user_location in user_locations:
-            location_type = user_location.location_type
-            expand_from = location_type.expand_from or location_type
-            expand_to_level = (SQLLocation.active_objects
-                               .filter(domain__exact=user.domain, location_type=location_type.expand_to)
-                               .values_list('level', flat=True)
-                               .first())
+    return LocationSet(get_location_fixture_queryset(user))
```

Comment:
I recommend reading this commit (Add in notes and general structure) with [`?w=1`](https://github.com/dimagi/commcare-hq/pull/16806/commits/354e0fc922d39f3412155e4f86ffec0db398ce7b?w=1)